Monika-After-Story/MonikaModDev
# Remove Upset Idle sprite

#3199 (sort of)

The upset idle sprite is now shared with the belowzero idle sprite. Idle-related dialogue (Talk + play menus) have been changed/removed/added so it fits better with the expression.

The belowzero idle sprite is a 95% `1esc` and 5% `5tsc`.

Since the idle sprite is the only visual change to upset, dialogue elsewhere should not need to be changed.

# Testing
1. set affection to upset (-74 to -30)
2. verify the idle sprite is same as belowzero (swapping between upset and belowzero normal will look the same) 
3. verify talk menu + play menu dialogue is appropriate for upset
